app/models/user_session.rb

```diff
@@ -11,7 +11,7 @@ def remove_step_ids(ids)
   end
 
   def import_date
-    return unless session['answers'].key?(Wizard::Steps::ImportDate.id)
+    return if session['answers'][Wizard::Steps::ImportDate.id].blank?
```

Comment:
The import date answer can be nil or an empty string. This covers both these variants.